_opensearch/cluster.md

```diff
@@ -12,23 +12,26 @@ OpenSearch can operate as a single-node or multi-node cluster. The steps to conf
 
 To create and deploy an OpenSearch cluster according to your requirements, it’s important to understand how node discovery and cluster formation work and what settings govern them.
 
-There are many ways to design a cluster. The following illustration shows a basic architecture:
+There are many ways to design a cluster. The following illustration shows a basic architecture that includes a four-node cluster that has one dedicated cluster manager node, one dedicated coordinating node, and two data nodes that are cluster manager-eligible and also used for ingesting data.
 
-![multi-node cluster architecture diagram]({{site.url}}{{site.baseurl}}/images/cluster.png)
+  **Note:** The nomenclature recently changed for the master node, it is now called the cluster manager node.
+   {: .tip }
 
-This is a four-node cluster that has one dedicated master node, one dedicated coordinating node, and two data nodes that are master-eligible and also used for ingesting data.
+![multi-node cluster architecture diagram]({{site.url}}{{site.baseurl}}/images/cluster.v2.png)
```

Comment:
Is the image misnamed here? I see that the image file `cluster.png`'s name didn't change, but we're linking to a v2 here.

Reply:
thanks! good catch. I forgot to update the reference to the 2nd version of the image file. ( i had previously created a separate png file, then thought better to just replace, so the diff between images would be easier to see for reviewers)